plugins/secret-squirrel/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@dotcom-tool-kit/secret-squirrel",
+  "version": "0.1.0",
+  "description": "",
+  "main": "lib",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "FT.com Platforms Team <platforms-team.customer-products@ft.com>",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/financial-times/dotcom-tool-kit.git",
+    "directory": "plugins/secret-squirrel"
+  },
+  "bugs": "https://github.com/financial-times/dotcom-tool-kit/issues",
+  "homepage": "https://github.com/financial-times/dotcom-tool-kit/tree/main/plugins/secret-squirrel",
+  "files": [
+    "/lib",
+    ".toolkitrc.yml"
+  ],
+  "peerDependencies": {
+    "@financial-times/secret-squirrel": "^2.19.0",
```

Comment:
**suggestion**: we should make peer dependency ranges as wide as possible: https://nodejs.org/en/blog/npm/peer-dependencies/#using-peer-dependencies
```suggestion
    "@financial-times/secret-squirrel": "2.x",
```